dev/TabView/TabView.idl

```diff
@@ -37,14 +37,21 @@ unsealed runtimeclass TabView : Windows.UI.Xaml.Controls.ListView
     Object RightCustomContent{ get; set; };
     Windows.UI.Xaml.DataTemplate RightCustomContentTemplate{ get; set; };
 
+    [MUX_DEFAULT_VALUE("true")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean IsAddButtonVisible{ get; set; };
+
     event Windows.Foundation.TypedEventHandler<TabView, TabViewTabClosingEventArgs> TabClosing;
 
+    event Windows.Foundation.TypedEventHandler<TabView, Object> AddButtonClick;
```

Comment:
>Object [](start = 56, length = 6)

Why Object?  Recommend having an EventArgs type so that you can version this over time to have other arguments in the future.  Object can never be versioned usefully.

Reply:
We used to make empty types so they could be versioned later, but I thought we've moved away from that -- again, I'll wait for API review to point us in the right direction.